packages/twenty-front/src/modules/workflow/workflow-trigger/utils/getCronTriggerDefaultSettings.ts

```diff
@@ -0,0 +1,44 @@
+import { CronTriggerInterval } from '@/workflow/workflow-trigger/constants/CronTriggerIntervalOptions';
+import { WorkflowCronTrigger } from '@/workflow/types/Workflow';
+
+const DEFAULT_CRON_PATTERN = '0 0 */1 * * *'; // Every hour
+
+export const getCronTriggerDefaultSettings = (
+  cronTriggerInterval: CronTriggerInterval,
+): WorkflowCronTrigger['settings'] => {
+  switch (cronTriggerInterval) {
+    case 'HOURS':
+      return {
+        schedule: {
+          hour: 1,
+          minute: 0,
+        },
+        type: cronTriggerInterval,
+        outputSchema: {},
+      };
+    case 'MINUTES':
+      return {
+        schedule: {
+          minute: 1,
+        },
+        type: cronTriggerInterval,
+        outputSchema: {},
+      };
+    case 'SECONDS':
+      return {
+        schedule: {
+          second: 30,
+        },
+        type: cronTriggerInterval,
+        outputSchema: {},
+      };
+    case 'CUSTOM':
+      return {
+        pattern: DEFAULT_CRON_PATTERN,
+        type: cronTriggerInterval,
+        outputSchema: {},
+      };
+    default:
+      throw new Error('Invalid cron trigger interval');
```

Comment:
You can use `assertUnreachable` to get a TypeScript error at build time if another interval is added but not covered in this function.

```suggestion
      return assertUnreachable(cronTriggerInterval);
```